rust/src/templates/newvendor.html

```diff
@@ -0,0 +1,115 @@
+
+<html>
+	<head>
+        <title>Local-and-Open</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+	</head>
+	<body>
+<div>
+    <div class="card text-center" style="width: 80%;">
+        <div class="card-header">
+            Shop eintragen
+        </div>
+
+    <div class="card-body">
+        <div class="card-subtitle mb-2 text-muted">
+           Aufnahme von ihrem Shop in die Datenbank
+        </div>
+
+        <div class=card-text>
+            Bitte f&uuml;llen Sie die folgenden Felder aus,
+            damit wir Ihren Shop entsprechend listen k&ouml;nnen.
```

Comment:
Haven't seen escaped Umlauts in a loooong time. :)

Reply:
and i don't know better ;)